general/solomiia_tymoshchuk/Task1.java

```diff
@@ -0,0 +1,53 @@
+package general.solomiia_tymoshchuk;
+
+import java.util.Scanner;
+
+/**
+ * Write a program that accepts numbers of miles as a floating point number,
+ * converts it to kilometers and outputs the result.
+ **/
+
+public class Task1 {
+    double kms;
+    String sentence;
+
+    public double getKms() {
+        return kms;
+    }
+
+    public String getSentence() {
+        return sentence;
+    }
+
+    public String setSentence() {
+        return sentence;
+    }
+
+    public String enterDataSentence(String sentence) {
+        System.out.println(sentence);
+        return sentence;
+    }
+
+    public Double convertKmstoMiles(double kms) {
+        double milesValue = 1.609;
+        double miles = kms * milesValue;
+        System.out.println(miles);
```

Comment:
I think it would be better to move output statements to the main method - you can use the returned value from this method.